### ISSUE TITLE
Files Not Converting to Ogg

### ISSUE THREAD

**Issue by floor32** [TeamTernate/infinite-music-discs#52]
On Windows 10, v1.2 of the datapack generator doesn't seem to be able to convert from mp3. I could be doing something wrong since I only have used it once but converting the file into ogg myself worked fine. 

Not really a big issue as it worked fine afterwards, except that some of the ogg converters I have tried haven't worked with the datapack and have only found one that works by one by one. If you or anybody seeing this has any file converter reccomendations lmk, not bothered if the generator doesn't do it.

Great app :)

**Comment by floor32** [comment]
Issues seems to be occuring from trying to convert from stereo to mono and not the file type

**Comment by bradytheinventor** [comment]
Hmm, I can convert any random mp3 from stereo to mono no problem.

Where did you get the original mp3 file? (Was it from a free youtube-to-mp3 converter?) What do you mean by it's not able to convert from mp3, is there an error message, is the resulting file actually still stereo? Did the app generate any log files?

Can you try attaching your problem mp3 file to this issue? You should be able to drag-drop it into the comment dialog.

**Comment by floor32** [comment]
The first file I ever used was an mp3 file which was from discord and pretty sure was not from youtube. It came up with the error not able to convert file to ogg or something along those lines. I then converted it using this website https://audio.online-convert.com/convert-to-ogg which worked fine when not converting from stereo to mono, but produced the same error when trying to convert within the app (which I should've done on the website but ah well). It seems I can't attach the mp3 file however but could email it to you. 

I then downloaded about 20 songs from a youtube to mp3, which wouldn't convert to ogg in the app, and wouldn't work with https://www.nch.com.au/switch/index.html?ref=cj&cjevent=4697b371452711ed81ebeb8c0a1c0e11 after being converted into ogg and then into the app, only with the above website which I can only do one by one. Once doing that I still was only able to generate the pack in stereo as otherwise the could not convert to ogg error appeared. The error only appears briefly so not sure of exact wording.
